### DISCUSSION TITLE
Feedback about github projects

### DISCUSSION THREAD

**Discussion by nicitaacom** [community/community#68910] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I would like to sort my tasks in github projects by date - but now I don't see this button

![image](https://github.com/community/community/assets/39565703/ba053970-c10f-4078-8ebb-2c24a79a64cc)

**Comment by lidong-linvest21** [comment]
You can create date field first, and go to that field to sort descending or ascending.

**Reply by nicitaacom** [reply]
What?

Please provide steps to do it

**Reply by lidong-linvest21** [reply]
1. Maybe you can switch to table layout. 2. create a new field about date. 3. sort the date.

**Reply by nicitaacom** [reply]
![image](https://github.com/community/community/assets/39565703/06ca78c4-6f72-4d1b-b0fc-8ea4d543fad2)

![image](https://github.com/community/community/assets/39565703/79e496db-3e87-4e72-99a8-24e557b5111c)

Issue persists

Please provide more details with screenshots

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @nicitaacom , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by nicitaacom** [reply]
sure - thank you
